pulsar-broker-common/src/main/java/com/yahoo/pulsar/broker/ServiceConfiguration.java

```diff
@@ -291,12 +301,32 @@ public void setWebServicePortTls(int webServicePortTls) {
         this.webServicePortTls = webServicePortTls;
     }
 
-    public boolean isBindOnLocalhost() {
-        return bindOnLocalhost;
+    public String getBindAddress() {
```

Comment:
I was trying to avoid `throws` on this method as the caller would have to require `try / catch` everywhere where this is used. Unless we can settle on `RuntimeException` or `IllegalArgumentException`?